firestore/api/GetData/Program.cs

```diff
@@ -154,20 +154,13 @@ private static async Task GetMultipleDocs(string project)
             QuerySnapshot capitalQuerySnapshot = await capitalQuery.GetSnapshotAsync();
             foreach (DocumentSnapshot documentSnapshot in capitalQuerySnapshot.Documents)
             {
-                if (documentSnapshot.Exists)
-                {
-                    Console.WriteLine("Document data for {0} document:", documentSnapshot.Id);
-                    Dictionary<string, object> city = documentSnapshot.ToDictionary();
-                    foreach (KeyValuePair<string, object> pair in city)
-                    {
-                        Console.WriteLine("{0}: {1}", pair.Key, pair.Value);
-                    }
-                    Console.WriteLine("");
-                }
-                else
+                Console.WriteLine("Document data for {0} document:", documentSnapshot.Id);
```

Comment:
I don't know this API well.  Can you explain why it's unnecessary to check Exists in this scenario?

Reply:
These are document snapshots that have been returned from a query. Non-existent snapshots are only returned if you call GetDocumentSnapshot on a DocumentReference where the document doesn't exist.

Reply:
Sure.  

DocumentSnapshot is used to represent a document read from Firestore, but if you are reading a document directly, it may not exist, e.g.:
```
var docSnap = Tasks.Await(db.Document("nonexisting/document").GetAsync());
Console.Out.WriteLine(docSnap.Exists); // False
```

But when you execute any sort of _query_ against Firestore, you'll get a `QuerySnapshot` containing all of the `DocumentSnapshot`s of the matching documents.  Since a document must exist in order to match the query, it is unnecessary to check `DocumentSnapshot.Exists` on the matching documents.

And if you look at the docs, all of the other languages (other than C#) don't have an exists check for this sample: https://firebase.google.com/docs/firestore/query-data/get-data

Reply:
Thank you for clarifying, and thank you for improving the sample.